lib/runner/extract-runnable-items.js

```diff
@@ -186,6 +188,15 @@ var sdk = require('postman-collection'),
 
         items = findItemsOrGroups(collection, entrypointLookup, true);
 
+        // Extracting the items and folders by the order in which they appear as folder/request arguments,
+        // only if specified in the entrypoint.preserveOrder
+        if (preserveOrder && items && items.reference && Array.isArray(entrypoints)) {
```

Comment:
```suggestion
        if (preserveOrder) {
```

- `Array.isArray(entrypoints)` - Already checked at `L179`
- `items && items.reference` - `findItemsOrGroups` is a internal function, guarantees to return `{ members, reference }`